src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyout.xaml.cs

```diff
@@ -274,6 +274,8 @@ private void IdentifierTextBox_KeyDown(object sender, KeyEventArgs e)
                         e.Handled = true;
                     }
                 }
+                // When user starts typing, stop to ask advice
+                smartRenameViewModel.Cancel();
```

Comment:
cc @AmadeusW  This will cancel the `cancellationToken` passed into `_smartRenameSession.GetSuggestionsAsync`
Also it will call the `OnCancel` method.